oci-build-examples/oci-build-caching/README.md

```diff
@@ -0,0 +1,80 @@
+# OCI DevOps - Speed up builds with caching
+
+Most of the time the build time is consumed in downloading or preparing certain build dependancies like maven dependencies, node_modules, pip dependancies, etc... This document talks about on how to reduce time by caching those dependencies for the subsequent runs. 
+
```

Comment:
Instead of document - its a illustration or procedure.

Reply:
Done